app/src/Index.re

```diff
@@ -1 +1,5 @@
+open Css;
+
+global("body", [color(Particle.Color.black()), ...Particle.Font.text]);
```

Comment:
I'd move this to App as we need those styles no matter where the app is applied. For me `Index.re` is the place which determines that we mount the app for a website served on some url and could conceive a different entry point for something like a desktop native deployment.

Reply:
Good point, I haven't yet figured out what's the best way to handle global stuff. Will move it into App for now.